packages/host/app/services/realm-info-service.ts

```diff
@@ -14,9 +14,7 @@ export default class RealmInfoService extends Service {
       return this.cachedRealmURLsForFileURL.get(fileURL)!;
     }
 
-    let response = await this.loaderService.loader.fetch(fileURL, {
-      headers: { Accept: SupportedMimeType.CardSource },
```

Comment:
It seems that this was not the issue breaking test altho I thought it was at one point so I just reverted here

Reply:
this seems like a bad assumption. how do you know that URL that is being requested is for card-source? This service is called "realm-info-service" it don't seem to indicate that this is only for card sources. Perhaps the accept header should be part of the parameters

Reply:
if I were to use the realmInfo provider, I'd need to do a fetch for card source. I can use a different method and a different cache so we will not make this bad assumption. But the method that is called `fetchExtension` can be used for CardSource

Reply:
having these implicit requirements on the callers feels like it could be the source of bugs if people don't clearly understand how to use this API. I would suggest that just make the accept header explicit by requiring callers to specify it when they provide the URL to get the realm info for. the accept header is part of how you ask for resources from the realm server--let;s just make that clear in our API for this service